### PR TITLE
Add more functions in test mod

### PR DESCRIPTION
**Pull request by QuocTriTran**
:)

### CODE REVIEW

File: server/src/parse/tests.rs [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+                        Conditions::Leaf(Condition {
+                            aliascol: None,
+                            col: "lname".to_string(),
+                            op: CompType::Equ,
+                            aliasrhs: None,
+                            rhs: CondType::Literal(Lit::Str("peng".to_string())),
+                            }
+                        )
+                    ),
+                        Box::new(
```
Comment (SvantjeJung) [inline server/src/parse/tests.rs]: Check indentation

File: server/src/parse/tests.rs [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+                                Conditions::Leaf(Condition {
+                                    aliascol: None,
+                                    col: "fname".to_string(),
+                                    op: CompType::Equ,
+                                    aliasrhs: None,
+                                    rhs: CondType::Literal(Lit::Str("peter".to_string())),
+                                    }
+                                )
+                            ),
+                                Box::new(Conditions::Leaf(Condition{
```
Comment (SvantjeJung) [inline server/src/parse/tests.rs]: Check indentation

File: server/src/parse/tests.rs [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+use std::collections::HashMap;
+
+// ============================================================================
+// Result::Ok unittest
+// ============================================================================
+
+#[test]
+fn test_create_table_empty() {
+    let mut p = parser::Parser::create("cReAtE table
+        foo");
```
Comment (SvantjeJung) [inline server/src/parse/tests.rs:22]: Fits in one row? Or is it seperated on purpose?

File: server/src/parse/tests.rs [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+            cols: Vec::<ColumnInfo>::new()
+        })))));
+}
+
+#[test]
+fn test_create_table_content() {
+    let mut p = parser::Parser::create(
+        "create table foo (FirstName char(255), LastName char(255))");
+
+    let vec = vec![ColumnInfo  {
```
Comment (SvantjeJung) [inline server/src/parse/tests.rs]: Two spaces between ColumnInfo and {

File: server/src/parse/tests.rs (hunk 3 above)
Comment (QuocTriTran) [inline server/src/parse/tests.rs:22]: It's for checking whether the parser can recognize "\n"

Comment (LukasKalbertodt) [inline server/src/parse/tests.rs:22]: You know... you can put `\n` into a Rust string literal? That way it's more clear what you want to test ;)
Currently it looks like an accidental line break ^_^

**Pull request merged**